pkg/azurefile/controllerserver.go

```diff
@@ -229,3 +277,31 @@ func (d *Driver) ListSnapshots(ctx context.Context, req *csi.ListSnapshotsReques
 func (d *Driver) ControllerExpandVolume(ctx context.Context, req *csi.ControllerExpandVolumeRequest) (*csi.ControllerExpandVolumeResponse, error) {
 	return nil, status.Error(codes.Unimplemented, "ControllerExpandVolume is not yet implemented")
 }
+
+func (d *Driver) getShareUrl(sourceVolumeID string) (azfile.ShareURL, string, error) {
```

Comment:
add comments for this func, explain what's ShareURL, also provide an example showing what s shareURL looks like